server/app/app.test.js

```diff
@@ -172,7 +172,8 @@ describe('Endpoints', () => {
 
   describe('PATCH /item/:id', () => {
     const updatedItem = {
-      name: "yeezys"
+      name: "yeezys",
+      reserve: true
     };
```

Comment:
Update Item name should only change the name

Reply:
Individualized each test to only test a single change at a time.